scripts/build

```diff
@@ -55,7 +77,7 @@ esac
 
 function rmStep(){
   rm -rf "$@"
-  rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
+  #rmdir -p --ignore-fail-on-non-empty `dirname "$@"`
```

Comment:
Odd - Haven't had to use rmdir right after a `rm -rf`. Gives errors on OSX related to the `--ignore-fail-on-non-empty` flag